Kinvey/Kinvey/Persistable.swift

```diff
@@ -93,6 +94,40 @@ public func <- <Transform: TransformType>(left: inout Transform.Object!, right:
     kinveyMappingType(left: right.0, right: right.1.currentKey!)
     left <- (right.1, right.2)
 }
+  
+class ListValueTransform<T: RealmSwift.Object>: TransformOf<List<T>, [JsonDictionary]> where T: BaseMappable {
```

Comment:
I think we should prefer using `TransformOf` to do quick transformations as described [here](https://github.com/Hearst-DD/ObjectMapper#transformof). 
If we need to extend (such as what you're doing here), `TransformType` seems more appropriate.